src/components/markdown/ProblemsList/ProblemListItemSolution.tsx

```diff
@@ -18,7 +18,7 @@ export default function ProblemListItemSolution(
     contents = (
       <div className="px-4 py-2 text-sm text-gray-300 dark:text-gray-500">
         <Tooltip
-          content={`We haven't written a solution for this problem yet. If needed, request one using the "Contact Us" button!`}
+          content={`We haven't written a solution for this problem yet.`}
```

Comment:
not sure if i should replace with something like "make an issue or email us"?

Reply:
sure let's replace it

Reply:
🤔 do links work in tooltips?